worker/dockerworker/dockerworker.go

```diff
@@ -27,6 +27,27 @@ type dockerworker struct {
 	cmd       *exec.Cmd
 }
 
+// See the meaning of worker location here
+// https://github.com/taskcluster/taskcluster-rfcs/pull/148/files
+func getWorkerLocation(cloudName string, workerConfig *cfg.WorkerConfig) (string, error) {
+	workerLocation := map[string]string{
+		"cloud": cloudName,
+	}
+
+	for _, key := range []string{"instanceId", "region", "instanceType"} {
+		if val, err := workerConfig.Get(key); err == nil {
+			workerLocation[key] = val.(string)
+		}
+	}
+
+	workerLocationEnv, err := json.Marshal(workerLocation)
+	if err != nil {
+		return "", err
+	}
+
+	return string(workerLocationEnv), nil
+}
```

Comment:
We'll need to set the same value for generic-worker and any other worker -- it's not docker-worker specific.  Probably this value should be calculated by each provider and put in a single state variable, `WorkerLocation`.  Then the workers can use this value.